datasetinsights/io/kfp_output.py

```diff
@@ -68,3 +76,15 @@ def write_metric(self):
         logger.debug(
             f"Metrics file {self.filename} saved at path:" f" {self.filepath}"
         )
+
+    def create_tb_visualization_json(self):
+        try:
+            metadata = {
+                "outputs": [{"type": "tensorboard", "source": self.tb_log_dir}]
+            }
+            with open("/mlpipeline-ui-metadata.json", "w") as f:
```

Comment:
@Saurav-D Why is this path hard coded? Can user specify this in CLI so that they can parameterize this in Kubeflow pipeline?

Reply:
This needs to be hardcoded because kubeflow wants it to be `/mlpipeline-ui-metadata.json` it does not works if the path is changed.